ui/backend/client_ui_postgres.py

```diff
@@ -162,3 +167,33 @@ def get_files_summary(self, repo_url):
                 " FROM discoveries WHERE repo_url=%s"
                 " GROUP BY file_name"
             ))
+
+    def add_embeddings(self, repo_url):
```

Comment:
again copy-pasted code from a parent class

Reply:
not exactly copy pasted, it's the same issue as update_similar_discoveries, there's the get_discoveries[1], that I can't get rid of